src/js/components/charts/Chart.js

```diff
@@ -47,15 +42,12 @@ var Chart = createReactClass({
       this.updateWidth();
     }
     global.addEventListener("resize", this.updateWidth);
-  },
+  }
 
   componentWillUnmount() {
+    this._mounted = false;
     global.removeEventListener("resize", this.updateWidth);
-  },
-
-  onSidebarStoreWidthChange() {
```

Comment:
we should put this one back ⚠️

Reply:
I couldn't find a caller at the time but now I see it's store listener magic